ferrocene/doc/user-manual/src/overview.rst

```diff
@@ -35,7 +39,7 @@ Further Relevant Documentation
 The Ferrocene documentation package includes:
 
 * The :doc:`specification:index`, which describes the expected behavior of Rust
-  as implemented by the Ferrocene compiler.
+  as implemented by the rustc compiler.
```

Comment:
would just leave line 42 out, else it reads like "describes behavior of Rust as implemented by Rust"

Reply:
What would you think about following?

> the expected behavior of Rust as implemented by rustc.